### DISCUSSION TITLE
Option for template variables to limit the possible values to only provided list

### DISCUSSION THREAD

**Discussion by molsza** [grafana/grafana#42782] [Dashboards, Panels & Visualizations]
With a template variable it is possible to provide a list of possible options, which will be presented as drop down lists. 

However nothing prevents the user to put any value in the select box and it will be used in dashboard's queries. 

The request is to add an option to variable template configuration, so grafana will not allow (or use) value which are not on the list. 

![demo](https://user-images.githubusercontent.com/41799777/144850855-f1d880c8-8f17-4a20-b0fe-7d8aeac29c13.gif)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
